### DISCUSSION TITLE
Plans for liboqs 0.11.0 release

### DISCUSSION THREAD

**Discussion by dstebila** [open-quantum-safe/liboqs#1858] [General]
https://github.com/open-quantum-safe/liboqs/milestone/25 is tracking major items that could be included in the liboqs 0.11.0 release.  As of this writing, it looks like we anticipate having #1745 resolved soon, and defer #1596 to a future release.  Is there anything else anyone things should be included in the release?  There is the possibility that a resolution to https://github.com/open-quantum-safe/liboqs/issues/1206 will also be available.

It might be possible to have a release in August based on current progress.

**Comment by dstebila** [comment]
Tying into [Roadmap discussion](https://github.com/orgs/open-quantum-safe/discussions/1892)

**Comment by SWilson4** [comment]
Relevant for 0.11.0: https://github.com/orgs/open-quantum-safe/discussions/1892#discussioncomment-10560897
